src/index.ts

```diff
@@ -413,13 +396,15 @@ export class GalleryComponent extends BaseComponent {
       thumb.initialHeight = medianHeight;
     }
 
-    this._$thumbs.link($.templates.galleryThumbsTemplate, this._thumbs);
+    const renderedHtml = this._thumbs.map(this._galleryThumbsTemplate).join('');
+    this._$thumbs.html(renderedHtml);

```

Comment:
Don't need link anymore https://github.com/BorisMoore/jquery-datalink?tab=readme-ov-file#jquerylink-api
Can just pass the full rendered html to jquery to apply to the dom